runtime/base/base.Dockerfile

```diff
@@ -244,7 +277,8 @@ RUN set -xe; \
     --enable-cookies \
     --with-gnu-ld \
     --with-ssl \
-    --with-libssh2
+    --with-libssh2 \
+    --with-nghttp2=${INSTALL_DIR}
```

Comment:
@mnapoli Ooops I also forgot to include that. It's all good now & ready to merge :angel:

Reply:
Could that be replaced by `--with-nghttp2`?

Because all the other flags don't have the path, I think this is because of the `--prefix=${INSTALL_DIR}` option above.

Reply:
This is a good question, and I've tried but it fail without the directory path so I had no other choice :sob: 

The only thing I haven't tried yet is to set some env vars, namely `CMAKE_INCLUDE_PATH` and `CMAKE_LIBRARY_PATH` as explained in https://github.com/curl/curl/pull/922#issuecomment-244625230.
Maybe it can work with that.

Reply:
Ok sorry I've double checked and it's OK we can remove it indeed :angel: